eng/pipelines/templates/steps/fetch-namespaces.yml

```diff
@@ -0,0 +1,29 @@
+parameters: 
+  DotnetNupkgLocation: ''
+  ArtifactName: ''
+  TargetBranch: ''
+  DocRepoLocation: ''
+  TargetDocRepoName: ''
+  TargetDocRepoOwner: ''
```

Comment:
Why not just default this to `Azure` and don't pass it above?

Reply:
Right. Good catch